### ISSUE TITLE
TrackEventStuck

### ISSUE THREAD

**Issue by Mora140** [Weeb-Devs/Laffey#54]
This usually happends when the bot stays like 12h on and then you try to play an youtube song , how can I fix this?

**Comment by Takiyo0** [comment]
what is the `payload` when the stuck even emitted? This can be because of your lavalink's server. But i never heard of this issue before

**Comment by Mora140** [comment]
Hello , I usually play an youtube song .. and the lavalink server is something s host free lavalink host (it commes free with discord bot host)..  I only changed the lavalink settings in .ENV and config.json file , I did it right?

**Comment by Takiyo0** [comment]
yes you did it right, and since it's a free host, sadly issue like this is common because the host is being used by a lot of people and the host's resource (cpu, network) must be affected.
